src/components/info-modal/component.js

```diff
@@ -0,0 +1,112 @@
+import React, { PureComponent, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Modal from 'components/modal';
+import MediaQuery from 'react-responsive';
+import { breakpoints } from 'utils/responsive';
+import HighlightedPlaces from 'components/widget/templates/highlighted-places/component';
+import highlightedPlacesConfig from 'components/widget/templates/highlighted-places/config';
+import LocationsList from 'components/locations-list';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import styles from './style.module.scss';
+
+class InfoModal extends PureComponent {
+  static propTypes = {
+    isOpened: PropTypes.bool,
+    currentLocation: PropTypes.shape({
+      name: PropTypes.string
+    }),
+    locations: PropTypes.arrayOf(PropTypes.shape({})),
+    highlightedPlaces: PropTypes.arrayOf(PropTypes.shape({})),
+    closeInfoPanel: PropTypes.func
+  }
+
+  static defaultProps = {
+    isOpened: false,
+    currentLocation: { name: 'Location name' },
+    locations: [],
+    highlightedPlaces: null,
+    closeInfoPanel: () => null
+  }
+
+  state = {
+    searchTerm: null
```

Comment:
Why is not a good idea to put this in the redux store?

Reply:
I don't think it's neccessary, we won't use it outside the component